scripts/build_antithesis_images.sh

```diff
@@ -74,6 +74,18 @@ function build_images {
     docker_cmd="${docker_cmd} --build-arg AVALANCHEGO_NODE_IMAGE=antithesis-avalanchego-node:${TAG}"
   fi
 
+  if [[ "${test_setup}" == "avalanchego" ]]; then
+    # Build the image that enables compiling golang binaries for the node and workload
+    # image builds. The builder image is intended to enable building instrumented binaries
+    # if built on amd64 and non-instrumented binaries if built on arm64.
+    #
+    # The builder image is not intended to be pushed so it needs to be built in advance of
+    # adding `--push` to docker_cmd. Since it is never prefixed with `[registry]/[repo]`,
+    # attempting to push will result in an error like `unauthorized: access token has
+    # insufficient scopes`.
+    ${docker_cmd} -t "${builder_image_name}" -f "${builder_dockerfile}" "${AVALANCHE_PATH}"
+  fi
```

Comment:
lgtm, would it make sense to add error handling? 
```suggestion
    if ! ${docker_cmd} -t "${builder_image_name}" -f "${builder_dockerfile}" "${AVALANCHE_PATH}"; then
      echo "Error: Failed to build builder image ${builder_image_name} with Dockerfile ${builder_dockerfile}"
      exit 1
    fi
  fi
```